src/hammer-vlsi/defaults.yml

```diff
@@ -348,9 +356,21 @@ vlsi.inputs:
     #  - no_connect (Optional[bool]) - If True the bump will be present but unconnected
     #  - x (Decimal) - The X coordinate of the bump assigned to this net
     #  - y (Decimal) - The Y coordinate of the bump assigned to this net
+    #  - group (Optional[str]) - The group name for the bumps, e.g. for creating schematic symbols
```

Comment:
Perhaps you can show me what this would do in person because I still don't get it.

Reply:
What was the conclusion from this? Could we add it to this documentation?